logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/EventMapper.kt

```diff
@@ -87,6 +87,22 @@ class EventMapper(
             is EventContentDTO.UserProperty.PropertiesDeleteDTO -> deleteUserProperties(id, eventContentDTO, transient)
             is EventContentDTO.Conversation.ReceiptModeUpdate -> conversationReceiptModeUpdate(id, eventContentDTO, transient)
             is EventContentDTO.Conversation.MessageTimerUpdate -> conversationMessageTimerUpdate(id, eventContentDTO, transient)
+            is EventContentDTO.Federation -> federation(id, eventContentDTO)
+        }
+
+    private fun federation(id: String, eventContentDTO: EventContentDTO.Federation): Event =
```

Comment:
Maybe we could use an explicit name here, something like `federationStopped` or `federationTerminated`

Reply:
I was thinking that some other events may come for federation but for now we can rename it 👍